### PR TITLE
index: Fix typos

### PR DESCRIPTION
**Pull request by mkhl**
Changes:
* projects was singular
* connection between microservices and projects was unclear
* dependencies on other modules was unnecessarily singular
* be was repeated
* work is not copyrighted but licensed under a license
* link to cc-by-sa was missing

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ experience in particular with microservices and
 [Self-contained Systems](http://scs-architecture.org) and the
 challenges faced in those projects.
 
-Far too often microservices are adopted but project fail to
+Far too often microservices are adopted but the projects then fail to
```
Comment (mkhl) [inline index.md]: I’m still not too happy with this. Maybe this?

> …are adopted in projects that then fail to succeed.

It makes the connection between the projects and microservices clearer, but it eliminates the “but” which I liked. :-/

**Comment by ewolff** [comment]
Excellent - thanks! :-)

File: index.md (hunk 1 above)
Comment (ewolff) [reply index.md]: I like both. Not sure which one is better.

Comment (mkhl) [reply index.md]: Let’s stick with “but” then, emphasizing that the projects fail in spite of using microservices.

**Pull request merged**